yammbs/_minimize.py

```diff
@@ -181,6 +181,9 @@ def _run_openmm(
         except UnassignedValenceError:
             logger.warning(f"Skipping record {qcarchive_id} with unassigned valence terms")
             return None
+        except ValueError as e:  # charging error
```

Comment:
IIRC, we'd want a more useful error but we can't as a downstream consequence of the toolkit's registry "let's try this method of every wrapper until it fails" design, correct?

Reply:
I think so? I was trying to just catch here what the toolkit was raising, which was... version 0.15.2, so it has possibly updated since then. IIRC the toolkit allows `ChargeCalculationError`s and so on to slip by when using `ToolkitRegistry.call` and then raises a single ValueError that bundles them up.

Reply:
For better or worse it hasn't changed